index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>First Mobile Page</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    
+    <header class="first">
+        <img src="/images/Welcome.png" alt="welcome button at the top left">
+        <img src="/images/Union.png" alt="a union of three vectors">
+    </header>
+    <br>
+    <div class="second">
+      <h1>  Hey there. I’m Badru
+I’m a software developer</h1>
```

Comment:
- [ ] you should stick to the Figma design and try to add the style provided there to this `<h1>` header tag in order to match the design 
- Figma design 
<img width="335" alt="Screen Shot 2022-09-21 at 18 24 27" src="https://user-images.githubusercontent.com/61790587/191570867-90c37b39-68e6-42c5-92e2-90135c99b4d3.png">
<img width="149" alt="Screen Shot 2022-09-21 at 18 24 39" src="https://user-images.githubusercontent.com/61790587/191571087-7ffd79e8-96b3-44cd-835b-a133c4e82375.png">
- Yours 
<img width="322" alt="Screen Shot 2022-09-21 at 18 25 34" src="https://user-images.githubusercontent.com/61790587/191571121-c37b9d5b-a7c9-4992-bcad-abe3f623ebdb.png">